base/src/main/java/uk/ac/ebi/atlas/solr/analytics/fullanalytics/AnalyticsQueryBuilder.java

```diff
@@ -0,0 +1,104 @@
+package uk.ac.ebi.atlas.solr.analytics.fullanalytics;
+
+import com.google.common.collect.Sets;
+import org.apache.solr.client.solrj.SolrQuery;
+import org.apache.solr.client.solrj.util.ClientUtils;
+
+import java.text.MessageFormat;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public class AnalyticsQueryBuilder {
+    enum Field {
+        EXPERIMENT_ACCESSION("experiment_accession"),
+        BIOENTITY_IDENTIFIER("bioentity_identifier"),
+        ASSAY_GROUP_ID("assay_group_id"),
+        EXPRESSION_LEVEL("expression_level");
+        final String name;
+
+        Field(String name){
+            this.name = name;
+        }
+
+        @Override
+        public String toString() {
+            return this.name;
+        }
+    }
+
+    private static final String SIMPLE_QUERY_TEMPLATE = "{0}:\"{1}\"";
```

Comment:
I like this class!

As far as I see you're only using these constants in one place each, you could controversially put them there and save everyone some scrolling.